db/structure.sql

```diff
@@ -359,87 +350,84 @@ CREATE SEQUENCE public.point_history_seq
 
 
 --
--- Name: point_history; Type: TABLE; Schema: public; Owner: -
+-- Name: point_histories; Type: TABLE; Schema: public; Owner: -
 --
 
-CREATE TABLE public.point_history (
+CREATE TABLE public.point_histories (
     id bigint DEFAULT nextval('public.point_history_seq'::regclass) NOT NULL,
-    usr_id bigint NOT NULL,
+    user_id bigint NOT NULL,
     point integer NOT NULL,
     semester character varying(9) NOT NULL
 );
 
 
 --
--- Name: points_2017; Type: TABLE; Schema: public; Owner: -
---
-
-CREATE TABLE public.points_2017 (
-    usr_firstname text,
-    usr_lastname text,
-    usr_nickname text,
-    usr_neptun character varying,
-    usr_bme_id character varying,
-    point integer
-);
-
-
---
--- Name: pontigenyles_id_seq; Type: SEQUENCE; Schema: public; Owner: -
+-- Name: point_requests_id_seq; Type: SEQUENCE; Schema: public; Owner: -
 --
 
-CREATE SEQUENCE public.pontigenyles_id_seq
+CREATE SEQUENCE public.point_requests_id_seq
     START WITH 1
     INCREMENT BY 1
     NO MINVALUE
     NO MAXVALUE
     CACHE 1;
 
 
-SET default_with_oids = true;
+--
+-- Name: point_requests; Type: TABLE; Schema: public; Owner: -
+--
+
+CREATE TABLE public.point_requests (
+    id bigint DEFAULT nextval('public.point_requests_id_seq'::regclass) NOT NULL,
+    point integer,
+    evaluation_id bigint NOT NULL,
+    user_id bigint
+);
+
 
 --
--- Name: pontigenyles; Type: TABLE; Schema: public; Owner: -
+-- Name: points_2017; Type: TABLE; Schema: public; Owner: -
 --
 
-CREATE TABLE public.pontigenyles (
-    id bigint DEFAULT nextval('public.pontigenyles_id_seq'::regclass) NOT NULL,
-    pont integer,
-    ertekeles_id bigint NOT NULL,
-    usr_id bigint
+CREATE TABLE public.points_2017 (
+    usr_firstname text,
+    usr_lastname text,
+    usr_nickname text,
+    usr_neptun character varying,
+    usr_bme_id character varying,
+    point integer
 );
 
 
 --
--- Name: poszt_seq; Type: SEQUENCE; Schema: public; Owner: -
+-- Name: poszttipus_seq; Type: SEQUENCE; Schema: public; Owner: -
 --
 
-CREATE SEQUENCE public.poszt_seq
+CREATE SEQUENCE public.poszttipus_seq
     START WITH 1
     INCREMENT BY 1
     NO MINVALUE
     NO MAXVALUE
     CACHE 1;
 
 
-SET default_with_oids = false;
-
 --
--- Name: poszt; Type: TABLE; Schema: public; Owner: -
+-- Name: post_types; Type: TABLE; Schema: public; Owner: -
 --
 
-CREATE TABLE public.poszt (
-    id bigint DEFAULT nextval('public.poszt_seq'::regclass) NOT NULL,
-    grp_member_id bigint,
-    pttip_id bigint
+CREATE TABLE public.post_types (
+    id bigint DEFAULT nextval('public.poszttipus_seq'::regclass) NOT NULL,
+    group_id bigint,
+    name character varying(30) NOT NULL,
+    delegated_post boolean DEFAULT false
 );
 
 
 --
--- Name: poszttipus_seq; Type: SEQUENCE; Schema: public; Owner: -
+-- Name: poszt_seq; Type: SEQUENCE; Schema: public; Owner: -
 --
 
-CREATE SEQUENCE public.poszttipus_seq
+CREATE SEQUENCE public.poszt_seq
```

Comment:
How does it become to `poszt`?

Reply:
These two lines didn't change, they are just swapped. (Check [your next comment](https://github.com/kir-dev/pek-next/pull/146#discussion_r373834304)) As I mentioned I just renamed the table/attribute names at first to remove all the `alias_attribute`/manually set table names and I didn't want to touch any other part of the db, because there are a lot of things to be changed/removed and also this is already a huge PR.

Reply:
Oh, I wasn't checked it correctly but you are right its just a reordering. 
And that is why I think that if we change a version, that it has to be separated.

And yes, its already enough big this PR.